Add Codespaces reference to README (#14)
TeamDev-IP/DotNetBrowser-QuickStart: README.md

This repository contains the development container (devcontainer) configuration that can be used to easily create a Linux-based development environment and work with DotNetBrowser there.

> You can use [GitHub Codespaces](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=489381301) or any of the [supported tools](https://containers.dev/supporting) to create a devcontainer and work with it. After the container is ready and the post-create command is executed, you can use the steps from the section above to build and run an example:

Author: ```suggestion
You can use [GitHub Codespaces](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=489381301) or any of the [supported tools](https://containers.dev/supporting) to create a devcontainer and work with it. After the container is ready and the post-create command is executed, you can build and run the console example:
```
